app/grandchallenge/components/models.py

```diff
@@ -570,7 +570,8 @@ def create_instance(self, *, image=None, value=None, fileobj=None):
         if image:
             civ.image = image
         elif fileobj:
-            civ.file = fileobj
+            container = File(fileobj)
+            civ.file.save(self.relative_path, container)
```

Comment:
Only keep the filename otherwise this will go into subfolders for nested interfaces:

```suggestion
            civ.file.save(Path(self.relative_path).name, container)
```